classes/class-wc-connect-paypal-ec.php

```diff
@@ -221,5 +226,30 @@ public function maybe_set_reroute_requests() {
 
 			wp_safe_redirect( wc_gateway_ppec()->get_admin_setting_link() );
 		}
-    }
+
+		public function update_order_status() {
+			$orders = wc_get_orders( array( 'status' => 'on-hold' ) );
+			error_log( count( $orders ) );
+
+			foreach ( $orders as $order ) {
+				$old_wc         = version_compare( WC_VERSION, '3.0', '<' );
+				$order_id       = $old_wc ? $order->id : $order->get_id();
+				$payment_method = $old_wc ? $order->payment_method : $order->get_payment_method();
```

Comment:
We have a compatibility layer to encapsulate WC version differences here: https://github.com/Automattic/woocommerce-services/blob/master/classes/class-wc-connect-compatibility.php. Here's an example use: https://github.com/Automattic/woocommerce-services/blob/212442410e1e9fd493c28473346efcab2cf2ddb0/classes/class-wc-connect-shipping-label.php#L52